source/_components/light.niko_home_control.markdown

```diff
@@ -0,0 +1,26 @@
+---
+layout: page
+title: "Niko Home Control"
+description: "Instructions on how to integrate Niko Home Control into Home Assistant."
+date: 2018-04-08 08:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_iot_class: "Local Polling"
+ha_release: 0.36
+---
+
+The `niko_home_control` platform allows you to integrate your [Niko Home Control](https://www.niko.eu/enus/products/niko-home-control) into Home Assistant.
+
```

Comment:
It would be nice if you would describe less briefly what types of devices are supported / what a potential user of the component/platform can expect to have with this platform.